docs/changelogs/CHANGELOG-2.24.md

```diff
@@ -5,6 +5,28 @@
 - [v2.24.2](#v2242)
 - [v2.24.3](#v2243)
 - [v2.24.4](#v2244)
+- [v2.24.5](#v2245)
+
+## [v2.24.5](https://github.com/kubermatic/kubermatic/releases/tag/v2.24.5)
+
+### Bugfixes
+
+- Add images for Velero and KubeLB to mirrored images list ([#13198](https://github.com/kubermatic/kubermatic/pull/13198))
+- Exclude `test` folders which contain symlinks that break once the archive is untarred ([#13151](https://github.com/kubermatic/kubermatic/pull/13151))
+- Fix a bug where OSPs were not being listed for VMware Cloud Director ([#6592](https://github.com/kubermatic/dashboard/pull/6592))
+- Fix a bug where dedicated vSphere credentials were incorrectly being required as mandatory input when editing provider settings for a cluster ([#6567](https://github.com/kubermatic/dashboard/pull/6567))
+- Fix invalid project ID in API requests for Nutanix provider ([#6572](https://github.com/kubermatic/dashboard/pull/6572))
+- Fix missing image registry override for hubble-ui components if Cilium is deployed as System Application ([#13139](https://github.com/kubermatic/kubermatic/pull/13139))
+- Fix: usercluster-controller-manager failed to reconcile cluster with disable CSI drivers ([#13183](https://github.com/kubermatic/kubermatic/pull/13183))
+- Update Azure CCM component version to v1.28.5 for user clusters with Kubernetes v1.28 from v1.28.0 address few LoadBalancer related issues found in some customer implementations ([#13173](https://github.com/kubermatic/kubermatic/pull/13173))
```

Comment:
Can be updated as follows --> `Fix Azure loadbalancer related issues found in some customer implementations by updating Azure CCM component version from v1.28.0 to v1.28.5 for the user clusters created with Kubernetes v1.28`